src/Console/ConfigurationResolver.php

```diff
@@ -479,7 +479,7 @@ public function getUsingCache(): bool
             }
         }
```

Comment:
in raised issue we have mentioned this docs:
> The cache is supported only when the tool was downloaded as a phar file or installed via Composer.

probably we can update it in this PR to reflect Docker image too